.github/dependabot.yml

```
@@ -0,0 +1,13 @@
+# Set update schedule for GitHub Actions
+
+version: 2
+updates:
+
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      # Check for updates to GitHub Actions every weekday
+      interval: "daily"
```

```suggestion
      interval: "weekly"
      day: "tuesday"
```
Checking weekly should suffice.